src/ng/directive/select.js

```diff
@@ -200,11 +200,13 @@ var selectDirective = function() {
 
         // we have to do it on each watch since ngModel watches reference, but
         // we need to work of an array, so we need to see if anything was inserted/removed
+        var lastView, lastViewRef = NaN;
```

Comment:
True - now that I see it, isn't it a bit weird that it is defined outside the link fn? So every selectDirective uses the same variable?